integrations/corporate-data-center.md

```diff
@@ -8,9 +8,14 @@ keywords: [on-premise, integrations]
 
 import IntegrationPrereqsCloud from "versionContent/_partials/_integration-prereqs-cloud-only.mdx";
 import TransitGateway from "versionContent/_partials/_transit-gateway.mdx";
+import NotSupportedAzure from "versionContent/_partials/_not-supported-for-azure.mdx";
 
 # Integrate your data center with $CLOUD_LONG
 
+<Tabs label="Tiger Cloud on AWS and Azure" persistKey="tiger-platform-clouds">
+
+<Tab title="Tiger Cloud on AWS" label="aws-cloud">
```

Comment:
I wasn't sure what that feature means, but when I read it turned out it's a Transit Gateway peering. It looks strange how it's called and that it's in the integrations section. It's in general, not related to Azure support.